documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -65,6 +65,19 @@ provided configuration. The possible statuses of the cache health are now HEALTH
 
 The Kryo and Protostuff marshallers have been deprecated and will be removed in Infinispan 15.0
 
+=== {brandname} now uses a version of the ProtoStream API that includes a bug
```

Comment:
The latest version won't include the bug :wink:

Reply:
Maybe I misinterpreted this: "But since getEnclosingClass() always returns null that leads to all types wrongly becoming top level types in the generated schema starting with 4.3.0 when this bug appeared."

ISPN 12 upgrades to 4.3.4 which is the version where the bug appears, isn't that so?

Reply:
> ISPN 12 upgrades to 4.3.4 which is the version where the bug appears, isn't that so?

No, 4.3.4 introduces a fix to a bug, but fixing that bug changes behaviour.

Reply:
OK. Should be good now. Thanks for bearing with me and explaining the details.